app/static/win32/github.sh

```diff
@@ -2,8 +2,27 @@
 
 CONTENTS="$(dirname "$(dirname "$(dirname "$(dirname "$(realpath "$0")")")")")"
 ELECTRON="$CONTENTS/GitHubDesktop.exe"
-CLI="$CONTENTS/Resources/app/cli.js"
+
+if grep -q Microsoft /proc/version; then
+	if [ -x /bin/wslpath ]; then
+		# On recent WSL builds, we just need to set WSLENV so that
+		# ELECTRON_RUN_AS_NODE is visible to the win32 process
+		export WSLENV=ELECTRON_RUN_AS_NODE/w:$WSLENV
+		CLI=$(wslpath -m "$CONTENTS/resources/app/cli.js")
+	else
+		# If running under older WSL, don't pass cli.js to Electron as
+		# environment vars cannot be transferred from WSL to Windows
+		# See: https://github.com/Microsoft/BashOnWindows/issues/1363
+		#      https://github.com/Microsoft/BashOnWindows/issues/1494
+		"$ELECTRON" "$@"
+		exit $?
+	fi
+elif [ "$(expr substr $(uname -s) 1 9)" == "CYGWIN_NT" ]; then
```

Comment:
Could you please change `==` to `=` as it causes the following message in WSL2 (Ubuntu):

```
dennis@DESKTOP-HIGHLVU:/mnt/c/Users/denni$ github
/mnt/c/Users/denni/AppData/Local/GitHubDesktop/bin/../app-2.4.1-beta1/resources/app/static/github.sh: 20: [: Linux: unexpected operator
```

When I changed it to `=` [as per these instructions](https://stackoverflow.com/a/3411105/4337765), I didn't get the warning anymore.

Reply:
done! :)